context/app/markdown/docs/metadata.md

```diff
@@ -1,39 +1,51 @@
-# Metadata
+## Metadata
 
 HuBMAP metadata documentation and submission specifications
 
-## Donor
+**Donor**
```

Comment:
No -`##` is better syntax for headings.

Reply:
@ngehlenborg @mccalluc Please go ahead with these changes